scripts/travis-ci.sh

```diff
@@ -20,8 +20,15 @@ travis_finish() {
   echo -en "\ntravis_time:end:$travis_timer_id:start=$travis_start_time,finish=$travis_end_time,duration=$duration\r\033[0m"
 }
 
+PARALLEL=-j2
+if [[ "$DIST" == "trusty" ]]; then
+    PARALLEL_CTEST=-j1
+else
+    PARALLEL_CTEST=-j4
```

Comment:
You may be able to use `nproc` to lookup the number of cores instead of having to hard-code it. Something like `PARALLEL_CTEST="-j$(nproc)"`.

Reply:
This is mostly tuned for memory consumption on virtual hosts, so it's hard to find a good number

Reply:
@kintel, by the way do you know what the issue was that led to keeping -j1 for trusty?  
https://github.com/openscad/openscad/commit/a0d92731292da184c7490ab606c24e518841e8e4

t-paul said something to the effect that it had caused some intermittent test failures?

Reply:
I think there was something related to Xvnc - multiple X11 environment causing some weird errors. Could also have been GL drivers.

Reply:
@justbuchanan It can help in some cases to set more jobs than cores, to help make up wasted cpu time when jobs are blocking on i/o.  so using the exact # of cores is not necessarily the goal.  but i feel like the j8 might have been excessive.  j4 seemed more reasonable and (in my sample size of 1 build) at -j4, the ctest portion of the macos job did run fastest compared to other builds at -j2 and -j8

I tried to benchmark these to some degree, but it is admittedly difficult to measure with the kind of variation in build times that we see from travis-ci.org 

Also something that may be of interest is the: "Backlog macOS Builds for Open Source projects" chart on this page:  https://www.traviscistatus.com/
You can get an idea how overloaded their mac servers are throughout the day/week. This mostly affects how long you have to wait before the mac build even boots/begins, sometimes 30min up to an hour or so if the backlog is high.  But I'm not sure if there is any high correlation between this backlog and the actual job run times, once they actually have begun.